Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/HomeContests.tsx

```diff
@@ -15,6 +16,11 @@ const HomeContests = () => {
         actions: { getForHome },
     } = useHomeContests();
 
+    const contestsToDisplay = useCallback(
+        (contests: IIndexContestsType[]) => contests.slice(0, maxContestsToDisplay),
```

Comment:
Send count to BE and return exact contests from controller instead of loading all and then slice it on FE